tests/cupy_tests/linalg_tests/test_eigenvalue.py

```diff
@@ -27,14 +27,32 @@ def _get_hermitian(xp, a, UPLO):
     reason='eigensolver not added until ROCm 4.2.0')
 class TestEigenvalue:
 
-    @testing.for_all_dtypes(no_float16=True, no_complex=True)
+    @testing.for_all_dtypes()
```

Comment:
Testing real arrays in complex dtypes is somewhat redundant.  How about merging `test_eigh_complex`?
```python
if numpy.dtype(dtype).kind == c:
    a = xp.array([[1, 2j, 3], [4j, 5, 6j], [7, 8j, 9]], _dtype)
else:
    a = xp.array([[1, 0, 3], [0, 5, 0], [7, 0, 9]], _dtype)
```

Reply:
Agreed. Now, I'll make `test_eigh` and `test_eigh_complex` into one.